src/components/pagination/Pagination.vue

```diff
@@ -3,98 +3,103 @@
         <slot
             v-if="$scopedSlots.previous"
             name="previous"
-            :page="getPage(current - 1, {
-                disabled: !hasPrev,
-                class: 'pagination-previous',
-                'aria-label': ariaPreviousLabel
-        })">
+            :page="
+                getPage(current - 1, {
+                    disabled: !hasPrev,
+                    class: 'pagination-previous',
+                    'aria-label': ariaPreviousLabel,
+                })
+            "
+        >
             <b-icon
                 :icon="iconPrev"
                 :pack="iconPack"
                 both
-                aria-hidden="true"/>
+                aria-hidden="true" />
         </slot>
         <BPaginationButton
             v-else
             class="pagination-previous"
             :disabled="!hasPrev"
             :page="getPage(current - 1)"
-            :aria-label="ariaPreviousLabel">
+            :aria-label="ariaPreviousLabel"
+        >
             <b-icon
                 :icon="iconPrev"
                 :pack="iconPack"
                 both
-                aria-hidden="true"/>
+                aria-hidden="true" />
         </BPaginationButton>
         <slot
             v-if="$scopedSlots.next"
             name="next"
-            :page="getPage(current + 1, {
-                disabled: !hasNext,
-                class: 'pagination-next',
-                'aria-label': ariaNextLabel
-        })">
+            :page="
+                getPage(current + 1, {
+                    disabled: !hasNext,
+                    class: 'pagination-next',
+                    'aria-label': ariaNextLabel,
+                })
+            "
+        >
             <b-icon
                 :icon="iconNext"
                 :pack="iconPack"
                 both
-                aria-hidden="true"/>
+                aria-hidden="true" />
         </slot>
         <BPaginationButton
             v-else
             class="pagination-next"
             :disabled="!hasNext"
             :page="getPage(current + 1)"
-            :aria-label="ariaNextLabel">
+            :aria-label="ariaNextLabel"
+        >
             <b-icon
                 :icon="iconNext"
                 :pack="iconPack"
                 both
-                aria-hidden="true"/>
+                aria-hidden="true" />
         </BPaginationButton>
 
+        <b-numberinput
+            v-if="pageInput"
+            class="pagination-input"
+            min="1"
+            :max="pageCount"
+            :value="inputValue"
+            :controls="false"
+            @input.native="onInput"
```

Comment:
Why native? Input already emits a number

Reply:
Why not simply input?

Reply:
I can. and should just use <input>. I will change that to just an input, since I am just getting the event and then getting the value from the event.

Reply:
Sure no problem. I am changing it now.